rs/state_machine_tests/src/lib.rs

```diff
@@ -1349,6 +1371,17 @@ impl StateMachine {
             membership_version: subnet_record.clone(),
             context_version: subnet_record,
         };
+        let pool_refill_task = self.pool_refill_task.read().unwrap();
+        let pool_refill_task = pool_refill_task.as_ref().unwrap();
+        self.runtime.block_on(async move {
+            pool_refill_task
+                .refill_pool(
+                    POOL_BYTE_SIZE_SOFT_CAP,
+                    POOL_SLICE_BYTE_SIZE_MAX,
+                    registry_version,
+                )
+                .await;
+        });
```

Comment:
@alin-at-dfinity Could you please suggest what to do here to refill the certified slice pool?